index.js

```diff
@@ -1,18 +1,138 @@
-const readline = require('readline');
+const { exit } = require("process");
+const readline = require("readline");
 const rl = readline.createInterface(process.stdin, process.stdout);
 
+// prompt user with question func
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
+//Get a random number in a range
+function randomNumGen(min, max) {
+  return Math.floor(Math.random() * (max - min) + min);
+}
+
 start();
 
+// Makes array 0 to max
+function createArray(max) {
+  let gameArray = [];
+  for (let i = 1; i <= parseInt(max); i++) {
```

Comment:
no single name variables please. also, you use 'i' as a variable in another context elsewhere which may cause variable shadowing. comments to explain what is happening here would be ideal.